lib/foods.js

```diff
@@ -9,18 +9,12 @@ $(document).ready( function() {
 function setFoodListener() {
   $('#food-form').on('submit', function(event) {
     event.preventDefault();
-    let name = $('[name=food-name]').val();
-    let calories = $('[name=calories]').val();
-
-    $.ajax({
-      method: 'POST',
-      url: api + '/api/v1/foods',
-      data: {food: {name: name, calories: calories}},
-      success: function(data) {
-        alert('Added food!')
-        foodRows(data);
-      }
-    })
+    // let name = $.trim($('[name=food-name]').val());
+    // let calories = $.trim($('[name=calories]').val());
+    let name = $('[name=food-name]');
```

Comment:
if the variable is a jQuery object (which it is in this case) - then we typically name it with a $ in front of it - `$name` so it's more explicit!